.github/workflows/check_crate.yml

```diff
@@ -0,0 +1,81 @@
+name: Check Crates
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check_crates:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y jq
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - name: Read member paths from main Cargo.toml
+        id: member_paths
+        run: |
+          paths=$(grep -oP '(?<=members = \[).*(?=\])' Cargo.toml | tr -d '\n')
+          paths=$(echo "$paths" | sed 's/,/\n/g' | tr -d '"')
+          echo "::set-output name=paths::$paths"
```

Comment:
Let's instead get Cargo.toml files in the `implementations/rust/ockam` directory
```bash
paths=$(find implementations/rust/ockam -name Cargo.toml)
```

Reply:
Done